.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "trailingComma": "es5",
   "singleQuote": false,
-  "semi": true
+  "semi": true,
+  "quoteProps": "consistent"
```

Comment:
I was OK with the default configuration, only requiring quotes when needed. But let's give this a try.